common/api-review/generative-ai.api.md

```diff
@@ -378,6 +386,8 @@ export interface GenerateContentCandidate {
     finishMessage?: string;
     // (undocumented)
     finishReason?: FinishReason;
+    // Warning: (ae-forgotten-export) The symbol "GroundingMetadata" needs to be exported by the entry point index.d.ts
```

Comment:
This indicates you haven't publicly exported any of the new types. The entry point for types is in `types/index.ts` (https://github.com/google/generative-ai-js/blob/0d4fe068cf8deb30ddc14bc3d33446696219bea5/types/index.ts) - add a line: `export * from "./search-grounding";` to make sure all the types you added in that file will be exported from the entry point.

Reply:
Done